### PR TITLE
Improve help texts for new and init commands

### PR DESCRIPTION
**Pull request by parkavi11**
Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help [hunk 1]
```diff
@@ -45,11 +46,17 @@ DESCRIPTION
...
+            Create a package with the 'main' function and the 'main' test.
+               $ bal new <package-name> -t main
 
             Create a package with a Ballerina service.
                $ bal new <package-name> -t service
 
             Create a package with a Ballerina library.
                $ bal new <package-name> -t lib
 
+            Create a package with a Ballerina central module.
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help:60]: We dont need this change

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help [hunk 2]
```diff
@@ -45,11 +46,17 @@ DESCRIPTION
 
        EXAMPLES
             Create a package with the 'main' function.
-               $ bal new <package-name>
+                $ bal new <package-name>
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help:49]: Wrong indentation

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help [hunk 3]
```diff
@@ -4,6 +4,7 @@ NAME
 SYNOPSIS
        bal new <package-path>
        bal new <package-path> [-t |--template] <template-name>
+       bal new <package-path> [-t |--template] <central-module>
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help:7]: This is not required as well

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help [hunk 4]
```diff
@@ -47,9 +48,11 @@ DESCRIPTION
             Create a package with the 'main' function.
                $ bal new <package-name>
 
+            Create a package with the 'main' function and the 'main' test.
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help:51]:
```suggestion
            Create a package with the 'main' function with tests.
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help [hunk 5]
```diff
@@ -47,9 +48,11 @@ DESCRIPTION
...
+               $ bal new <package-name> -t main
+
             Create a package with a Ballerina service.
                $ bal new <package-name> -t service
 
             Create a package with a Ballerina library.
                $ bal new <package-name> -t lib
-
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help:55]: We dont need this change

**Comment by parkavi11** [comment]
Closing as this change is sent to `ballerina-platform:2.0.0-beta3-dev` branch

**Pull request closed without merging**